templates/ts-ttk/teamsapp.local.yml

```
@@ -0,0 +1,69 @@
+# yaml-language-server: $schema=https://aka.ms/teams-toolkit/1.0.0/yaml.schema.json
+# Visit https://aka.ms/teamsfx-v5.0-guide for details on this file
+# Visit https://aka.ms/teamsfx-actions for details on actions
+version: 1.0.0
+
+environmentFolderPath: ./env
+
+provision:
+
+  - uses: script
+    name: Increment manifest version
+    with:
+      run: node increment-version.js
+      workingDirectory: scripts
+
+  - uses: aadApp/create
+    with:
+      name: $\{{APP_NAME}}-$\{{TEAMSFX_ENV}}
+      generateClientSecret: true
+      signInAudience: AzureADMultipleOrgs
+    writeToEnvironmentFile:
+      clientId: AAD_APP_CLIENT_ID
+      clientSecret: SECRET_AAD_APP_CLIENT_SECRET
+      objectId: AAD_APP_OBJECT_ID
+      tenantId: AAD_APP_TENANT_ID
+      authority: AAD_APP_OAUTH_AUTHORITY
+      authorityHost: AAD_APP_OAUTH_AUTHORITY_HOST
+
+  - uses: aadApp/update
+    with:
+      manifestPath: ./aad.manifest.json
+      outputFilePath: ./build/aad.manifest.$\{{TEAMSFX_ENV}}.json
```

We should update rename this and move it to the infra folder.

```suggestion
      manifestPath: ./infra/entra/entra.manifest.json
      outputFilePath: ./infra/entra/build/entra.manifest.$\{{TEAMSFX_ENV}}.json
```